### DISCUSSION TITLE
Generating and plotting MLWF

### DISCUSSION THREAD

**Discussion by starofus10** [wannier-berri/wannier-berri#321] [Q&A]
Dear All,

I want to ask, is it possible to compute and generate MLWF using Wannierberri? 
In this case, I want to get the output .xsf file to plot the MLWF.

Best Regards
Joshua

**Comment by stepan-tsirkin** [comment]
What exactly do you mean? (1) computing the unitary rotation matrices which define the Wannier Gauge, or (2) having those matrices - compute them of a 3D grid

for (1) since recently there is a [disentanglement](https://github.com/wannier-berri/wannier-berri/blob/master/wannierberri/system/disentanglement.py) algorithm implemented , but not much tested and there is no maximal localization.  For (2) - no, that is not implemented here. 

What is the issue which do not satisfy you in Wannier90 for these functionalities?
